AcademyApp/Protocols/NibLoadableView.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  NibLoadableView.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 21/06/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol NibLoadableView: class {
```

Comment:
Markers + comments why to have this protocol

Reply:
Added markers + comments to protocol and extensions.